spec/lib/rmt/cli/export_spec.rb

```diff
@@ -167,17 +171,43 @@
           expect(mirror_double).to receive(:mirror).with(
             repository_url: 'http://foo.bar/repo1',
             auth_token: 'foobar',
-            local_path: '/repo1'
+            local_path: '/repo1',
+            do_not_raise: nil
           ).and_raise(RMT::Mirror::Exception, 'black mirror')
 
           expect(mirror_double).to receive(:mirror).with(
             repository_url: 'http://foo.bar/repo2',
             auth_token: '',
-            local_path: '/repo2'
+            local_path: '/repo2',
+            do_not_raise: nil
           )
 
           expect { command }.to output(/black mirror/).to_stderr
         end
+
+        # it 'does not raise exception' do
+        #   FileUtils.mkdir_p path
+        #   File.write("#{path}/repos.json", repo_settings.to_json)
+
+        #   expect(mirror_double).to receive(:mirror_suma_product_tree).with({ repository_url: 'https://scc.suse.com/suma/' })
+        #   expect(mirror_double).to receive(:mirror).with(
+        #     repository_url: 'http://foo.bar/repo1',
+        #     auth_token: 'foobar',
+        #     local_path: '/repo1',
+        #     do_not_raise: true
+        #   ).and_raise(RMT::Mirror::Exception, 'black mirror')
+
+        #   expect(mirror_double).to receive(:mirror).with(
+        #     repository_url: 'http://foo.bar/repo2',
+        #     auth_token: '',
+        #     local_path: '/repo2',
+        #     do_not_raise: true
+        #   )
+
+        #   expect { command }.to_not(
+        #     raise_error(RMT::Mirror::Exception, 'black mirror')
+        #   )
+        # end
```

Comment:
I guess that all this is to be reworked and I'm reviewing this too early, right? 😄